src/v/transform/rpc/serde.cc

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "transform/rpc/serde.h"
+
+#include "model/record.h"
+
+#include <seastar/core/chunked_fifo.hh>
+
+namespace transform::rpc {
+transformed_topic_data::transformed_topic_data(
+  model::topic_partition tp, model::record_batch b)
+  : tp(std::move(tp)) {
+    ss::chunked_fifo<model::record_batch> batches;
+    batches.reserve(1);
+    batches.push_back(std::move(b));
```

Comment:
question: Did you mean to declare a new instance of `batches` here?

Reply:
Nope :)